src/Common/hooks/useExport.tsx

```diff
@@ -22,7 +22,13 @@ export default function useExport() {
     return <CSVLink id={id} filename={filename} data={data} />;
   };
 
-  const getTimestamp = () => new Date().toISOString();
+  const getTimestamp = () => {
+    let d = new Date();
+    let date = d.toLocaleDateString();
+    let time = d.toLocaleTimeString();
+
+    return date+"_"+time;
+}
```

Comment:
Why is this here? Ensure changes from your other branches are not present in other PRs.

Reply:
ok, will do